### DISCUSSION TITLE
Announcing Official Discord Server!

### DISCUSSION THREAD

**Discussion by cabralpinto** [cabralpinto/modular-diffusion#11] [Announcements]
Hi everyone,

Exciting news! 🌟 We've hit 100 stars on our repo, and I couldn't be happier! We also got featured in [TLDR AI](https://tldr.tech/ai/2023-09-04)! Thank you so much for your support and all the awesome feedback. You rock! 🙌
 
During this time, a few of you have reached out on Reddit and LinkedIn to make questions about Modular Diffusion. To ensure I can assist everyone more effectively, I've gone ahead and launched a new Discord server!

🔗 Check it out here: https://discord.gg/mYJWQATfTV. Let's chat!

Cheers,
João

**Comment by sneccc** [comment]
can you send again link is broken, put to never expire 
![image](https://github.com/cabralpinto/modular-diffusion/assets/61990796/37429c51-72b1-4b78-b570-c317707e6a95)

**Reply by cabralpinto** [reply]
Thanks for the heads up! I'm not very well versed in Discord so I didn't know this was an issue. I'll fix it in the next 5 minutes.
